chapter-4/e3-a-list.js

```diff
@@ -0,0 +1,59 @@
+function arrayToList(arr) {
+  return addValueToList(arr, list);
```

Comment:
why have a list arg when it's just undefined? I think you should think through this design a bit more.

Reply:
Also, it's weird to have the return above the function definition